crates/matrix-sdk/src/room/common.rs

```diff
@@ -99,22 +104,68 @@ impl Common {
 
     /// Leave this room.
     ///
+    /// Returns a [`Result`] containing an instance of [`Left`] if successful.
+    ///
     /// Only invited and joined rooms can be left
-    pub(crate) async fn leave(&self) -> Result<()> {
+    #[doc = concat!("\n\n", include_str!(concat!(env!("CARGO_MANIFEST_DIR"), "/src/docs/sync_running.md")))]
```

Comment:
No need for `\n\n` concatenation if the file just starts with a newline.

Reply:
1. I tried that, but the newline did not show up in the docs. Then again, the current version did not work with just one `\n`, so maybe two newlines in the file will do? I'll check.